plugins/modules/aci_interface_policy_leaf_profile.py

```diff
@@ -202,48 +251,53 @@
 def main():
     argument_spec = aci_argument_spec()
     argument_spec.update(
-        leaf_interface_profile=dict(type='str', aliases=['name', 'leaf_interface_profile_name']),  # Not required for querying all objects
+        interface_profile=dict(type='str', aliases=['name', 'leaf_interface_profile_name', 'leaf_interface_profile']),  # Not required for querying all objects
         description=dict(type='str', aliases=['descr']),
         state=dict(type='str', default='present', choices=['absent', 'present', 'query']),
         name_alias=dict(type='str'),
+        type=dict(type='str', default='leaf', choices=['fex', 'leaf']),
     )
 
     module = AnsibleModule(
         argument_spec=argument_spec,
         supports_check_mode=True,
         required_if=[
-            ['state', 'absent', ['leaf_interface_profile']],
-            ['state', 'present', ['leaf_interface_profile']],
+            ['state', 'absent', ['interface_profile']],
+            ['state', 'present', ['interface_profile']],
         ],
     )
 
-    leaf_interface_profile = module.params.get('leaf_interface_profile')
+    interface_profile = module.params.get('interface_profile')
     description = module.params.get('description')
     state = module.params.get('state')
     name_alias = module.params.get('name_alias')
+    type_profile = module.params.get('type')
 
     aci = ACIModule(module)
+    class_rn = 'infraFexP/fexprof' if type_profile == 'fex' else 'infraAccPortP/accportprof'
```

Comment:
same, class rn does not seems to be used so directly assign two variable to avoid split.